doc/source/whatsnew/v0.24.1.rst

```diff
@@ -50,7 +50,7 @@ Bug Fixes
 
 **I/O**
 
--
+- Bug in :class:`HDFStore` that caused it to raise ``ValueError`` when reading a Dataframe in Python 3 from dataframe written in Python 2 (:issue:`24925`)
```

Comment:
I think we'll have you move this 0.24.2.

Reply:
The v0.24.2.rst document is not created (https://github.com/pandas-dev/pandas/tree/master/doc/source/whatsnew), so I added it to this ```v0.24.1.rst```. Moreover, I think that last release is v0.24.0. Am I right?